src/config.js

```diff
@@ -46,22 +52,24 @@ const conf = convict({
   db: {
     host: {
       doc: 'Hostname where PostgreSQL is running',
-      format: 'url',
       default: 'localhost'
     },
     port: {
       doc: 'Port where PostgreSQL is running',
-      format: 'port',
+      format: 'dbport',
       default: 5432
     },
     name: {
-      doc: 'Database name'
+      doc: 'Database name',
+      default: 'sensorweb',
     },
     user: {
-      doc: 'Database username'
+      doc: 'Database username',
+      default: '',
     },
     password: {
-      doc: 'Database password'
+      doc: 'Database password',
+      default: '',
```

Comment:
username and password arguments are always needed by the underlying subsystem, so let's have a simple default